common/djangoapps/student/models.py

```diff
@@ -281,6 +281,8 @@ class Meta(object):
     allow_certificate = models.BooleanField(default=1)
     bio = models.CharField(blank=True, null=True, max_length=3000, db_index=False)
     profile_image_uploaded_at = models.DateTimeField(null=True)
+    title = models.CharField(blank=True, null=True, max_length=256)
+    company = models.CharField(blank=True, null=True, max_length=256)
```

Comment:
I don't think we need to add these fields or the related migration. The extended registration fields should get saved to the auth_userprofile.meta field.